mfr/extensions/papaya/static/README-papaya.txt

```diff
@@ -0,0 +1,7 @@
+Currently using a custom built papaya.js with option to remove the ability to open new files from the users desktop.
```

Comment:
TODO: Remove this file. PR was accepted and we are no longer using custom build.

Reply:
:tada:  Nicely done!